daq/runner.py

```diff
@@ -259,9 +261,11 @@ def main_loop(self):
 
         try:
             monitor = stream_monitor.StreamMonitor(idle_handler=self._handle_system_idle,
-                                                   loop_hook=self._loop_hook)
+                                                   loop_hook=self._loop_hook,
+                                                   timeout_ms=20000) #Polling rate
```

Comment:
I would keep this constant timeout_sec for consistency with other uses, and then *1000 as necessary.

Reply:
This is the original timeout input on polling in stream monitor but it wasn't being used. Do you want me to change the name so it matches the other timeout_sec?